troubadix/plugins/script_calls_empty_values.py

```diff
@@ -57,12 +58,16 @@ def check_content(
                 plugin=self.name,
             )
 
-        matches = _get_special_script_tag_pattern(
-            name=r"(?!add_preferences).*", value=""
-        ).finditer(file_content)
-        for match in matches:
-            yield LinterError(
-                f"{match.group(0)} does not contain a value",
-                file=nasl_file,
-                plugin=self.name,
-            )
+        for call in SpecialScriptTag:
+            if call == SpecialScriptTag.ADD_PREFERENCE:
+                continue
+            # Special script tag name can not be a regex
```

Comment:
```suggestion
```

IMHO this could be also completely dropped.

Reply:
Removed